willeswa/politico-ui
# #163668616 creates a candidate running for office

#### What does this PR do?
Registers a new candidate for election
### Description of Tasks to be completed?
Register a user as a candidate running for a ​political office​. Accessible by
the admin alone
### How should this be manually tested?
By following the steps outlined in the repo,
clone the repo and send a POST request to the `/office/<office-id>/register` endpoint in postman
You should be able to create a new candidate in the database
### What are the relevant Pivotal Tracker stories?
https://www.pivotaltracker.com/story/show/163668616
